build.gradle

```diff
@@ -25,13 +25,13 @@ def javaVersion = VERSION_11
 
 // The CordApp section.
 // This is part of the DSL provided by the corda plugins to define metadata for our CorDapp.
-// Each component of the CorDapp would get its own CorDapp section in the build.gradle file for the component’s
+// Each component of the CorDapp gets its own CorDapp section in the build.gradle file for the component’s
 // subproject.
 // This is required by the corda plugins to build the CorDapp.
 cordapp {
     // "targetPlatformVersion" and "minimumPlatformVersion" are intended to specify the preferred
     // and earliest versions of the Corda platform that the CorDapp will run on respectively.
-    // Enforced versioning has not implemented yet so we need to pass in a dummy value for now.
+    // Enforced versioning has not been implemented yet, enter a dummy value for now.

```

Comment:
Change from:

Enforced versioning has not been implemented yet, enter a dummy value for now.

To:

Enforced versioning has not been implemented yet, so enter a dummy value for now.